.gitignore

```
@@ -0,0 +1,5 @@
+*.swp
+*.exe
+
```

Add a line for me? :)
```suggestion
.idea
```

---

Certainly :)